packages/dropdowns/src/elements/Dropdown/Dropdown.tsx

```diff
@@ -49,7 +51,8 @@ const Dropdown: React.FunctionComponent<IDropdownProps & ThemeProps<DefaultTheme
     onSelect,
     onStateChange,
     onInputValueChange,
-    downshiftProps
+    downshiftProps,
+    role = 'listbox'
```

Comment:
Should we use `defaultProps` to provide this value? That way it is shown in the prop-sheet.

Reply:
Yeah this sounds good 👍 .